refactoring/about.md

```diff
@@ -0,0 +1,9 @@
+# Refactoring
+
+Refactoring is very important in relation to these topics. Just as important as the other topics covered here, is how to take undesirable code and turn it into great code.
```

Comment:
> to these topics

Maybe mention them again? I.e., make explicit that refactoring is basically taking "bad" code and applying patterns and idioms (and domain knowledge and common sense) to make it better.